### PR TITLE
Info page

### PR DESCRIPTION
**Pull request by irisliu77**
This page includes some basic info about me.

### CODE REVIEW

**Comment by irisliu77** [comment]
I didn't notice that my push request this morning was not successful. I've repushed those that don't have the above problems. Sorry.

**Comment by joanchenx** [comment]
Like Andrew mentioned, I think you deleted the images used on your homepage. You should add those back in.

**Comment by irisliu77** [comment]
I delete those images in this branch because i only make change in "About Me" page in my website..

**Comment by irisliu77** [comment]
And i don't actually have those image at this branch locally..

**Comment by joanchenx** [comment]
By deleting the images in this branch, you're going to merge those deletions and remove the images from master.

In [this commit](https://github.com/irisliu77/STEP/pull/4/commits/e5073b922784740fdde2f1da2655b1a860ca863d), you deleted the files.

You'll want to add the images back so that they match what's in master. You can either upload the files again or I think this will work too: https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request

**Comment by joanchenx** [comment]
Approved :)

File: portfolio/src/main/webapp/me.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="node_modules/bulma/css/bulma.min.css" />
```
Comment (anbenson) [inline portfolio/src/main/webapp/me.html:6]: These tags don't need to have the trailing slash, and according to the Google HTML style guide they shouldn't have them.

**Pull request merged**